aio/content/examples/toh-pt2/src/app/heroes/heroes.component.ts

```diff
@@ -19,7 +19,7 @@ export class HeroesComponent implements OnInit {
   heroes = HEROES;
   // #enddocregion component
   // #docregion on-select
-  selectedHero: Hero;
+  selectedHero: Hero | undefined;
```

Comment:
I think it would be cleaner to go with

```suggestion
  selectedHero?: Hero;
```